<!-- loading lightning launch file, and set parameters -->
<launch>
  <rosparam ns="move_group" command="load" file="$(find lightning)/config/ompl_planning.yaml" />
  <include file="$(find lightning)/launch/lightning_neural.launch"/>
  <param name="/lightning/robot_name" value="baxter" />
  <param name="/lightning/env_name" value="baxter" />
  <param name="/lightning/planner_name" value="rrtstar" />
  <param name="/lightning/draw_points" value='false' />
  <param name="/lightning/planner_config_name" value="RRTStarkConfigDefault" />

  <rosparam command='load' file='$(find lightning)/scripts/experiments/baxter/baxter_model_parameters.yaml' ns='lightning/model' subst_value='True'>
    <!-- ROS parameters for Neural Network -->
  </rosparam>
</launch>
